src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.cs

```diff
@@ -46,6 +46,20 @@ public partial class TextBlock : DependencyObject
 
 		private Run _reusableRun;
 		private bool _skipInlinesChangedTextSetter;
+		private bool _subscribeToPointerEvents;
+		private bool _isPressed;
+		private (int start, int end) Selection
```

Comment:
`(int start, int end)` would benefit from being a record that can be referenced, such as `TextSpan` or similar.